src/result.cpp

```diff
@@ -13,7 +13,7 @@ using topgg::ratelimited;
 #pragma clang diagnostic ignored "-Wunused-function"
 #endif
 
-[[maybe_unused]] static const char* get_dpp_error_message(const dpp::http_error& http_error) {
+TOPGG_MAYBE_UNUSED static const char* get_dpp_error_message(const dpp::http_error& http_error) {
```

Comment:
any reason why you'd do this? [[maybe_unused]] is standard since before C++17 as far as I'm aware, is there any need to make yet more macros?

Reply:
ah, my instinct for C++ attributes was to make macros for them lmao, didn't know that it was standard since C++17, will fix!